engines/mock/mocksandbox.go

```diff
@@ -70,13 +74,29 @@ func (s *sandbox) AttachProxy(name string, handler http.Handler) error {
 	// Lock before we access proxies as this method may be called concurrently
 	s.Lock()
 	defer s.Unlock()
+	if strings.ContainsAny(name, " ") {
+		return engines.NewMalformedPayloadError("MockEngine proxy names cannot contain space")
+	}
```

Comment:
Yes... That would be better... This is just a mock engine... But yes lets do it anyway